assets/common/utils.gd

```diff
@@ -0,0 +1,4 @@
+class_name Utils
+
+static func push_log(arg1 = "", arg2 = "", arg3 = "", arg4 = "", arg5 = "", arg6 = "", arg7 = "", arg8 = ""):
```

Comment:
Is this the best way to do this? Is there nothing in gd script for a spread array for args?

Reply:
Yeah I spent ages looking into nicer or cleaner alternatives, to which I believe there is discussion on, but no agreement on implementation for variadic functions yet here: https://github.com/godotengine/godot-proposals/issues/1034.

Reply:
Realistically the issue is there's no matching `push_log` method to `push_warning` and `push_error` so we can keep `print` statements separate to logs.

Reply:
@BenKellySoftware I've gone and renamed the function to `push_info` to match the proposed function name.